src/jquery.i18n.fallbacks.js

```diff
@@ -1,4 +1,4 @@
-/*!
+/* !
```

Comment:
See http://eslint.org/docs/rules/lines-around-comment, you want it to be disabled?